src/modules/home/components/LoginButton.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react'
 import styles from '../styles/LoginButton.module.css'
+import { useNavigate } from 'react-router-dom'
+import { loginPath } from '../../login/routes/LoginRoute'
 
 export const LoginButton = () => {
+    const toLogin = useNavigate()
```

Comment:
This should be either one of two things

```
const navigate = useNavigate()
```
Then below you would create an onClick
```
const onClick = () => {
    navigate('loginPath')
}
```
Then the button receives
`onClick=onclick` 
as the prop

The reason i would never call it `const toLogin = useNavigate()`
Is if i pass `onClick={() => toLogin(registerPath)}`
It would now still work but i would go to register. So const toLogin, is not correct, its just navigation which takles a paramter which defines where it goes